src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -394,4 +394,10 @@
   <data name="InputFileNotSpecified" xml:space="preserve">
     <value>Please specify a nuspec, project.json, project file to use</value>
   </data>
+  <data name="Error_PackFailed" xml:space="preserve">
+    <value>Error packaging '{0}'.</value>
+  </data>
+  <data name="Error_InvalidTargetFramework" xml:space="preserve">
+    <value>Unsupported targetFramework value on '{0}'</value>
```

Comment:
Could this have more info, example:
    Packing your project failed because of an unsupported targetFramework "foo". The project file is "c:\foo\project.json".
Maybe not that exactly, but I'd like to know which target framework is bad and in which project, and what can't be done because of the invalid framework.